stdlib/contrib/jsternberg/rows/rows.flux

```diff
@@ -1,16 +1,70 @@
+// Package rows provides additional functions for remapping values in rows.
+//
+// introduced: 0.77.0
 package rows
 
 
-// map will map each of the rows to a new value.
-// The function will be invoked for each row and the
-// return value will be used as the values in the output
-// row.
-//
-// The record that is passed to the function will contain
-// all of the keys and values in the record including group
-// keys, but the group key cannot be changed. Attempts to
-// change the group key will be ignored.
-//
-// The returned record does not need to contain values that are
-// part of the group key.
+// map is an alternate implementation of [`map()`](https://docs.influxdata.com/flux/v0.x/stdlib/universe/map/)
+// that is faster, but more limited than map().
+// rows.map() cannot modify [groups keys](https://docs.influxdata.com/flux/v0.x/get-started/data-model/#group-key)
+// and, therefore, does not need to regroup tables.
+// **Attempts to change columns in the group key are ignored.**
+//
+// ## Parameters
+//
+// - fn: A single argument function to apply to each record.
+//   The return value must be a record.
+//   (Use the with operator to preserve columns not in the group
+//   and not explicitly mapped in the operation.)_
+//
+// ## Examples
+// ### Perform mathemtical operations on column values
+// The following example returns the square of each value in the _value column:
+//
+// ```
+// import "contrib/jsternberg/rows"
+//
+// data
+//   |> rows.map(fn: (r) => ({ _value: r._value * r._value }))
+// ```
+//
+// > Important notes
+// > The _time column is dropped because:
+// > It’s not in the group key.
+// > It’s not explicitly mapped in the operation.
+// > The with operator was not used to include existing columns.
```

Comment:
~~~suggestion
// The following example returns the square of each value in the `_value` column:
//
// ```
// import "contrib/jsternberg/rows"
// import "sampledata"
//
// < sampledata.int()
// >     |> rows.map(fn: (r) => ({ _value: r._value * r._value }))
// ```
//
// **Important**: The `_time` column is dropped because:
// 
// - It’s not in the group key.
// - It’s not explicitly mapped in the operation.
// - The `with` operator was not used to include existing columns.
```